Dockerfile

```diff
@@ -30,6 +30,7 @@ ENV HOME=/graph-explorer
 ENV NEPTUNE_NOTEBOOK=$NEPTUNE_NOTEBOOK
 RUN if [ -n "$NEPTUNE_NOTEBOOK" ] && [ "$NEPTUNE_NOTEBOOK" = "true" ]; then \
       echo "GRAPH_EXP_ENV_ROOT_FOLDER=/proxy/9250/explorer" >> ./packages/graph-explorer/.env; \
+      echo "PROXY_SERVER_HTTP_PORT=9250" >> ./packages/graph-explorer/.env; \
```

Comment:
Moving the proxy port setting to a single variable is a good change. Can we take this one step further, and also pull `PROXY_SERVER_HTTP_PORT` from a user-configurable env variable? The default should remain as 9250 if not supplied.

Reply:
@michaelnchin If I understand what you are suggesting, I think we are aligned here.

What I imagine is a future where the need for `NEPTUNE_NOTEBOOK` as specific argument to the docker container would disappear.

Instead, the notebook instances would launch the normal Docker container with the environment variables it needs defined.

Something like this:

```
docker run -p 9250:9250 \
 --env HOST={hostname-or-ip-address} \
 --env GRAPH_EXP_ENV_ROOT_FOLDER='/proxy/9250/explorer' \
 --env PROXY_SERVER_HTTP_PORT='9250' \
 -v /path/to/config.json:/graph-explorer/config.json \
 graph-explorer
```

Reply:
> What I imagine is a future where the need for NEPTUNE_NOTEBOOK as specific argument to the docker container would disappear.

IMO we should still keep `NEPTUNE_NOTEBOOK` available to ensure that the default port, root folder, and other static settings like the [HTTP/S options](https://github.com/aws/graph-explorer/blob/649be37370a7dd7768d6ee58aec505563b07b059/process-environment.sh#L23-L24) can be pinned down via a single convenient flag.

> --env GRAPH_EXP_ENV_ROOT_FOLDER='/proxy/9250/explorer' \ 
--env PROXY_SERVER_HTTP_PORT='9250' \

Seems ideal - we should add these variables to the existing settings picked up by [process-environment.sh](https://github.com/aws/graph-explorer/blob/main/process-environment.sh#L7-L16). User-supplied port and root folder values should override the defaults currently set based on the value/presence of `NEPTUNE_NOTEBOOK`. 

Current set of changes look good for now though, let's pick up the above in a future PR.